dbcrossbarlib/src/drivers/bigquery_shared/column.rs

```diff
@@ -58,7 +58,11 @@ impl BqColumn {
             BqDataType::NonArray(ty) => (ty, Mode::Required),
         };
         Ok(BqColumn {
-            name: col.name.replace('-', &'_'.to_string()).to_owned(),
+            name: col
+                .name
+                .replace('-', &'_'.to_string())
+                .replace(' ', &'_'.to_string())
```

Comment:
these could probably be combined with a regex - `[\-\s]+`

Reply:
@seamusabshere yeah. Updating

Reply:
Updated